app/components/topNav.tsx

```diff
@@ -0,0 +1,223 @@
+import logo_hori from "../imgs/SocialPlan-it-logo-Horizontal.png";
+import logo_sq from "../imgs/SocialPlan-it-logo-Square.png";
+
```

Comment:
Why don't move all images to the public folder?

Reply:
Great first move! What's the most important thing to consider with images and in what issues could we run when hosting our images as part of our code in the public folder?